docs/TheArtOfHttpScripting.md

```diff
@@ -456,6 +456,8 @@ SPDX-License-Identifier: curl
  it is time to set the User Agent field to fool the server into thinking you
  are one of those browsers.
 
+ By default, curl uses curl/VERSION, such as User-Agent: curl/`%VERSION`.
```

Comment:
The %VERSION variable expansion does not work in this document. It should probably spell out an example version verbatim.

Reply:
@bagder thank you for clarifying. Updated with hard-coded example version number.